products/billingbudget/api.yaml

```diff
@@ -172,16 +172,22 @@ objects:
             properties:
               - !ruby/object:Api::Type::String
                 name: pubsubTopic
-                required: true
                 description: |
                   The name of the Cloud Pub/Sub topic where budget related
                   messages will be published, in the form
                   projects/{project_id}/topics/{topic_id}. Updates are sent
                   at regular intervals to the topic.
               - !ruby/object:Api::Type::String
                 name: schemaVersion
-                default_value: "1.0"
                 description: |
                   The schema version of the notification. Only "1.0" is
                   accepted. It represents the JSON schema as defined in
                   https://cloud.google.com/billing/docs/how-to/budgets#notification_format.
+              - !ruby/object:Api::Type::Array
+                name: monitoringNotificationChannels
```

Comment:
Mind adding an example or test adding this field?

Reply:
Sure. I'll look into this shortly and add another commit to this PR.

Reply:
Done. Have added an example/test `templates/terraform/examples/billing_budget_notify.tf.erb`

Reply:
Thanks for the example, it looks great!